ci-operator/step-registry/ovn/conf/custom-cidr/ovn-conf-custom-cidr-ref.yaml

```diff
@@ -4,12 +4,12 @@ ref:
   commands: ovn-conf-custom-cidr-commands.sh
   resources:
     requests:
-      cpu: 10m
-      memory: 100Mi
+      cpu: 1000m
+      memory: 1Gi
```

Comment:
Hum I think this step itself just do a configuration change, not cpu and memory consuming. Please double check if this is necessary like the if the previous failure is directly related to this setting.